unison-share-api/src/Unison/Server/Local/Endpoints/Current.hs

```diff
@@ -0,0 +1,85 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE RecordWildCards #-}
+
+module Unison.Server.Local.Endpoints.Current where
+
+import Control.Monad.Except
+import Data.Aeson
+import Data.OpenApi (ToSchema (..))
+import Servant ((:>))
+import Servant.Docs (ToSample (..))
+import U.Codebase.Sqlite.DbId
+import U.Codebase.Sqlite.Queries qualified as Queries
+import U.Codebase.Sqlite.Project qualified as Project
+import U.Codebase.Sqlite.ProjectBranch qualified as ProjectBranch
+import Unison.Core.Project (ProjectAndBranch (..), ProjectName (..), ProjectBranchName (..))
+import Unison.Codebase (Codebase)
+import Unison.Codebase qualified as Codebase
+import Unison.Codebase.Path qualified as Path
+import Unison.Prelude
+import Unison.Server.Backend
+import Unison.Server.Types (APIGet)
+import Unison.Project.Util (pattern UUIDNameSegment)
+import Unison.NameSegment (NameSegment (..))
+
+type CurrentAPI =
+  "current"
```

Comment:
Could we move this to `ucm/current` (I'm not sure what the right syntax for doing so is)

Reply:
I'll go ahead and do that :)